build.gradle

```diff
@@ -1129,11 +1133,17 @@ buildDeb {
 }
 
 distTar {
+  archiveName = 'TIB_compute-ce_' + version + '_linux.tar.gz'
+  if (isEnterpriseProduct) {
+    archiveName = 'TIB_compute_' + version + '_linux.tar.gz'
+  }
```

Comment:
Instead of hardcoding linux, use osName to derive like in the ext block after osName declaration: osName.getFamilyName().replace(' ', '').toLowerCase()

Reply:
Done for this and distZip.